CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+
+**Running integration-tests manually**:
+
+    $ pytest --verbose --maxfail=1 --capture=no tests/
+
+When all the python dependencies have been installed, the integration test suite
+can also be invoked directly.
+
+The `maxfail` parameter will abort after the first failure, and `capture=no`
+will print the complete compiler output, and test log.
+
+**Running unit-tests manually**:
+
+    $ cmake -B build -D CMAKE_RUNTIME_OUTPUT_DIRECTORY=$(pwd)/build
+    $ cmake --build build --target sentry_test_unit
+    $ ./build/sentry_test_unit
+
+The unit-tests are a separate executable target and can be built and run on
+their own.
+
+## How to interpret CI failures
+
+The way that tests are run unfortunately does not make immediately obvious from
+the summary what the actual failure is, especially for compile-time failures.
+In such cases, it is good to scan the test output _from top to bottom_ and find
+the offending compile error.
+
+When running tests locally, one can use the `--maxfail=1` / `-x` parameter to
+abort after the first failure.
+
+## Integration Test Parameters
+
+The integration test suite runs the `sentry_example` target using a variety of
+different compile-time parameters, and asserts different use-cases.
+
+Some of its behavior is controlled by env-variables:
+
+- `ERROR_ON_WARNINGS`: Turns on `-Werror` for gcc compatible compilers.
+  This is also the default for `MSVC` on windows.
+- `RUN_ANALYZER`: Runs the code with/through one or more of the given analyzers.
+  This accepts a comma-separated list, and currently has support for:
+  - `asan`: Uses clangs AddressSanitizer and runs integration tests with the
+    `detect_leaks` flag.
+  - `scan-build`: Runs the build through the `scan-build` tool.
+  - `code-checker`: Uses the [`CodeChecker`](https://github.com/Ericsson/codechecker)
+    tool for builds.
+  - `kcov`: Uses [`kcov`](https://github.com/SimonKagstrom/kcov) to collect
+    code-coverage statistics.
+  - `gcc`: Use the `-fanalyzer` flag of `gcc > 10`.
+    This is currently not stable enough to use, as it leads to false positives
+    and internal compiler errors.
+- `TEST_X86`: Passes flags to CMake to enable a 32-bit (cross-)compile.
+- `ANDROID_API` / `ANDROID_NDK` / `ANDROID_ARCH`: Instructs the test runner to
+  build using the given Android `NDK` version, targeting the given `API` and
+  `ARCH`. The test runner assumes an already running simulator matching the
+  `ARCH`, and will run the tests on that.
+
+**Running examples manually**:
+
+    $ cmake -B build -D CMAKE_RUNTIME_OUTPUT_DIRECTORY=$(pwd)/build
+    $ cmake --build build --target sentry_example
+    $ ./build/sentry_example log capture-event
+
+The example can be run manually with a variety of flags to test different
+scenarios. Additionally, it will use the `SENTRY_DSN` env-variable, and can thus
+also be used to capture events/crashes directly to sentry.
+
+The example currently supports the following flags:
```

Comment:
Maybe clarify to pass those flags without a `--`, as this is what I expected in the past. Maybe, call them "commands" instead?